ontology/ontoavida.md

```diff
@@ -8,7 +8,9 @@ contact:
   github: miguelfortuna
   orcid: 0000-0002-8374-1941
 description: "OntoAvida develops an integrated vocabulary for the description of the most widely-used computational approach for studying evolution using digital organisms (i.e., self-replicating computer programs that evolve within a user-defined computational environment)."
-domain: digital evolution, a branch of Artificial Life.
+domain: other
+tags:
+  - digital evolution, a branch of Artificial Life.
```

Comment:
are these two different tags?

Reply:
Why not. digital evolution. artificial life